src/routes/anime/explore/+page.svelte

```diff
@@ -132,7 +131,7 @@
 				.join()
 		};
 		for (const [key, value] of Object.entries(search_map)) {
-			if (!is_empty(value)) {
+			if (value?.length) {
```

Comment:
No, this is super stupid.

value can be string too

```js
`hello`.length === 5
```

Reply:
> value can be string too

yes? it can be
i dont get it?